com.github.fries1234.ncsa-mosaic.json

```diff
@@ -0,0 +1,112 @@
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/motif/Motif%202.3.8%20Source%20Code/motif-2.3.8.tar.gz",
+                    "sha256": "859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7"
+
+                }
+            ],
+            "post-install": [
+                "rm -rf /app/lib/*.a"
+            ]
+        },
+        {
+            "name": "mosaic",
+            "buildsystem": "simple",
+            "build-commands": [
+                "CFLAGS=-fcommon meson build",
+                "ninja -C build",
+                "install -Dm755 build/src/Mosaic -t /app/bin",
+                "install -Dm644 com.github.fries1234.ncsa-mosaic.desktop /app/share/applications/com.github.fries1234.ncsa-mosaic.desktop",
+                "install -Dm644 com.github.fries1234.ncsa-mosaic.appdata.xml /app/share/metainfo/com.github.fries1234.ncsa-mosaic.appdata.xml",
+                "install -Dm644 com.github.fries1234.ncsa-mosaic-symbolic.svg /app/share/icons/hicolor/symbolic/apps/com.github.fries1234.ncsa-mosaic-symbolic.svg",
+                "install -Dm644 com.github.fries1234.ncsa-mosaic.png /app/share/icons/hicolor/256x256/apps/com.github.fries1234.ncsa-mosaic.png",
+                "install -Dm644 com.github.fries1234.ncsa-mosaic.128px.png /app/share/icons/hicolor/128x128/apps/com.github.fries1234.ncsa-mosaic.png",
+                "install -Dm644 com.github.fries1234.ncsa-mosaic.64px.png /app/share/icons/hicolor/64x64/apps/com.github.fries1234.ncsa-mosaic.png"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/fries1234/ncsa-mosaic",
+                    "branch": "master",
+                    "commit": "3fcf1f51568defd0e89b3c134f129634ad7dbbed"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.fries1234.ncsa-mosaic.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.fries1234.ncsa-mosaic.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.fries1234.ncsa-mosaic-symbolic.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.fries1234.ncsa-mosaic.png"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.fries1234.ncsa-mosaic.128px.png"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.fries1234.ncsa-mosaic.64px.png"
+                }
+            ],
+            "post-install": [
+                "rm -rf /app/bin/uil /app/bin/mwm /app/bin/xmbind /app/share/Xm /app/share/man /app/include/*"
+            ]
```

Comment:
you shoud use "cleanup" to remove uneeded files. Note that it's either at the top level, or per module.